BuildHelpers/Public/Get-BuildVariables.ps1

```diff
@@ -69,13 +69,13 @@ function Get-BuildVariables {
             }
             $true
         })]
-        $GitPath = 'git'
+        $GitPath
```

Comment:
The hard coded git.exe we are searching for makes this default value unnecessary.